scala-generator/src/test/scala/models/play/files/ModelsGensSpec.scala

```diff
@@ -58,18 +96,18 @@ class ModelsGensSpec extends FunSpec with Matchers with PropertyChecks {
   }
 
   it("generates play js value generator") {
-    forAll { (str: String) =>
+    forAll(genNsStr) { (str: String) =>
       val ns = scala.generator.Namespaces(str)
       val expected = s"""
         private[${ns.last}] ${ModelsGens.arbitrary(ns, "JsValue", ModelsGens.JsValue)}
-        private[${ns.last}] lazy val genJsValue: ${ModelsGens.Gen}[${ModelsGens.JsValue}] = ${ModelsGens.Gen}.lzy {
-          ${ModelsGens.Gen}.oneOf(
-            ${ModelsGens.Arbitrary}.arbitrary[IndexedSeq[${ModelsGens.JsValue}]].map(${ModelsGens.JsArray}),
-            ${ModelsGens.Arbitrary}.arbitrary[Boolean].map(${ModelsGens.JsBoolean}),
-            ${ModelsGens.Gen}.const(${ModelsGens.JsNull}),
-            ${ModelsGens.Arbitrary}.arbitrary[BigDecimal].map(${ModelsGens.JsNumber}),
-            // ${ModelsGens.Arbitrary}.arbitrary[${ModelsGens.JsObject}],
-            ${ModelsGens.Arbitrary}.arbitrary[String].map(${ModelsGens.JsString})
+        private[${ns.last}] lazy val genJsValue: _root_.org.scalacheck.Gen[${ModelsGens.JsValue}] = _root_.org.scalacheck.Gen.lzy {
+          _root_.org.scalacheck.Gen.oneOf(
+            _root_.org.scalacheck.Arbitrary.arbitrary[IndexedSeq[${ModelsGens.JsValue}]].map(${ModelsGens.JsArray}),
+            _root_.org.scalacheck.Arbitrary.arbitrary[Boolean].map(${ModelsGens.JsBoolean}),
+            _root_.org.scalacheck.Gen.const(${ModelsGens.JsNull}),
+            _root_.org.scalacheck.Arbitrary.arbitrary[BigDecimal].map(${ModelsGens.JsNumber}),
+            // _root_.org.scalacheck.Arbitrary.arbitrary[${ModelsGens.JsObject}],
```

Comment:
`// comment`

Reply:
Leaving untouched as it was commented out before